### PR TITLE
Update create-a-phone-system-call-queue.md

### PR DESCRIPTION
**Pull request by CLYVR**
Removing licensing information and placing references as it is now centralized in the planning article.  Updated information on external transfers for Overflow and Timeout.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-call-queue.md [hunk 1]
```diff
@@ -142,9 +137,9 @@ You can choose to disconnect the call or redirect it to any of the [call routing
 
 ![Screenshot of call timeout settings](media/call-queue-timeout-handling.png)
 
-**Call Timeout: maximum wait time** specifies the maximum time a call can be on hold in the queue before it is redirected or disconnected. You can specify a value from 15 seconds to 45 minutes.
+**Call Timeout: maximum wait time** specifies the maximum time a call can be on hold in the queue before it is redirected or disconnected. You can specify a value from 0 seconds to 45 minutes.
 
-You can choose to disconnect the call or redirect it to one of the call routing destinations. For example, you might have the caller leave a voicemail for the agents in the queue.
+You can choose to disconnect the call or redirect it to one of the call routing destinations. For example, you might have the caller leave a voicemail for the agents in the queue.For external transfers, please refer to [Prerequisites](plan-auto-attendant-call-queue.md#prerequisites) and the [external phone number transfers - technical details](create-a-phone-system-auto-attendant.md#external-phone-number-transfers---technical-details) for number formatting.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:142]:
```suggestion
You can choose to disconnect the call or redirect it to one of the call routing destinations. For example, you might have the caller leave a voicemail for the agents in the queue. For external transfers, please refer to [Prerequisites](plan-auto-attendant-call-queue.md#prerequisites) and the [external phone number transfers - technical details](create-a-phone-system-auto-attendant.md#external-phone-number-transfers---technical-details) for number formatting.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@msmikedavis PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**